.github/workflows/ci.yml

```diff
@@ -14,11 +14,13 @@ jobs:
         include:
         - java: 8
         - java: 11
+        - java: 17
     steps:
     - uses: actions/checkout@v2.3.4
-    - uses: olafurpg/setup-scala@v10
```

Comment:
Replace by `setup-java` as `setup-scala` is using Jabba, which is not maintained anymore

Reply:
`Setup-java` includes caching:
`cache: sbt`
Then you could remove the `rm` command, too